pyaedt/desktop.py

```diff
@@ -403,6 +403,8 @@ def _init_desktop(self):
     def _set_version(self, specified_version, student_version):
         version_student = False
         if specified_version:
+            if int(str(specified_version).split(".")[0])<2021:
```

Comment:
I think I could simplify this line. `specified_version` being a string.
```suggestion
            if float(specified_version)<2021:
```